pkg/proxy/proxyports/proxyports.go

```diff
@@ -304,6 +304,19 @@ func (p *ProxyPorts) AckProxyPortWithReference(ctx context.Context, name string)
 func (p *ProxyPorts) AckProxyPort(ctx context.Context, name string, pp *ProxyPort) error {
 	p.mutex.Lock()
 	defer p.mutex.Unlock()
+
+	if !pp.acknowledged {
```

Comment:
Just curios to know couple of things related to these changes:
1. Since the changes are adding logs for the scenarios where `pp.acknowledged` is `false`, should the same changes be added to the function `AckProxyPortWithReference` given they both call the `p.ackProxyPort` (which updates the `acknowledged` parameter ?
2. In what scenarios does this code gets executed ? (Just want to understand the use case for having these logs).